### PR TITLE
Add table variations and fix bug for lists inside tables

### PR DESCRIPTION
**Pull request by discodavey**

### CODE REVIEW

File: assets/sass/patterns/atoms/list.scss [hunk 1]
```diff
@@ -6,13 +6,6 @@
   padding-left: 0.75rem;
 }
 
-// Had to put this here as list.scss overrides table.scss
```
Comment (nlisgo) [inline assets/sass/patterns/atoms/list.scss:9]: Does this no longer apply?

File: source/_patterns/00-atoms/components/list.mustache [hunk 2]
```diff
@@ -17,3 +17,128 @@
     </ul>
 
 {{/isOrdered}}
+
+
+ <ul class="list">
```
Comment (nlisgo) [inline source/_patterns/00-atoms/components/list.mustache:22]: This should be removed

File: source/_patterns/00-atoms/components/table.mustache [hunk 3]
```diff
@@ -22,5 +22,4 @@
     </ol>
 
   {{/hasFootnotes}}
-
-</div>
+</table>
```
Comment (nlisgo) [inline source/_patterns/00-atoms/components/table.mustache:25]: There is no opening `</table>` in this template.

Comment (nlisgo) [reply source/_patterns/00-atoms/components/table.mustache:25]: I think this should be reverted.

**Review by nlisgo — CHANGES_REQUESTED** [review]
I've added comments.

File: assets/sass/patterns/atoms/list.scss (hunk 1 above)
Comment (discodavey) [reply assets/sass/patterns/atoms/list.scss:9]: Have you got the latest version as I removed this on Friday?

**Review by nlisgo — APPROVED** [review]

**Pull request merged**